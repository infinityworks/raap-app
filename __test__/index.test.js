/* eslint-env jest */

import React from "react";
import renderer from "react-test-renderer";
import { Index } from "../pages/index.js";

const defaultProps = {
  createTodo: () => {},
  subscribeToNewTodos: () => {},
  todos: [],
}

describe("With Snapshot Testing", () => {
  it('Index shows index page"', () => {
    const component = renderer.create(<Index {...defaultProps}  />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
