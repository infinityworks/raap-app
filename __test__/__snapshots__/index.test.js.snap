// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Index shows index page" 1`] = `
<div>
  <p>
    Hello World
  </p>
  <input
    onChange={[Function]}
    value=""
  />
  <button
    onClick={[Function]}
  >
    Create Todo
  </button>
  <ul>
    <li>
      <a
        href="/post/Hello Next.js"
        onClick={[Function]}
      >
        Hello Next.js
         TEST
      </a>
    </li>
    <li>
      <a
        href="/post/Learn Next.js is awesome"
        onClick={[Function]}
      >
        Learn Next.js is awesome
         TEST
      </a>
    </li>
    <li>
      <a
        href="/post/Deploy Indexs with Zeit"
        onClick={[Function]}
      >
        Deploy Indexs with Zeit
         TEST
      </a>
    </li>
  </ul>
</div>
`;
